cassandra-analytics-release/finish_release.sh

```diff
@@ -0,0 +1,226 @@
+rm _tmp_msg_
+
+#
+# Determine deb/rpm repo series
+#
+
+idx=`expr index "$release" -`
+if [ $idx -eq 0 ]
+then
+    release_short=${release}
+else
+    release_short=${release:0:$((idx-1))}
+fi
+release_major=$(echo ${release_short} | cut -d '.' -f 1)
+release_minor=$(echo ${release_short} | cut -d '.' -f 2)
+repo_series="${release_major}${release_minor}x"
+
+# Cleaning up
+execute "cd $cassandra_dir"
+rm -rf $tmp_dir
+
+# Restore stdout/stderr (and close temporary descriptors) if not verbose
+[ $verbose -eq 1 ] || exec 1>&3 3>&- 2>&4 4>&-
+
+mail_file="$mail_dir/mail_release_$release"
+[ ! -e "$mail_file" ] || rm $mail_file
+
+echo "[RELEASE] Apache Cassandra Analytics $release released" > $mail_file
+echo "" >> $mail_file
+echo "The Cassandra team is pleased to announce the release of Apache Analytics Cassandra version $release." >> $mail_file
+echo "" >> $mail_file
+echo "" >> $mail_file
+echo "Downloads of source and binary distributions are available here:" >> $mail_file
+echo "" >> $mail_file
+echo " https://dlcdn.apache.org/cassandra/cassandra-analytics/$release/" >> $mail_file
+echo "" >> $mail_file
+echo "" >> $mail_file
+echo "The Maven artifacts can be found at: " >> $mail_file
+echo "" >> $mail_file
+echo " https://repo.maven.apache.org/maven2/org/apache/cassandra/" >> $mail_file
+echo "" >> $mail_file
+echo "These will be mirrored to other repositories." >> $mail_file
+echo "" >> $mail_file
+echo "" >> $mail_file
+echo "As always, please review the changes[1] and pay attention to the release notes[2]. Let us know[3] if you were to encounter any problem." >> $mail_file
+echo "" >> $mail_file
+echo "" >> $mail_file
+echo 'Enjoy!' >> $mail_file
+echo "" >> $mail_file
+echo "[1]: CHANGES.txt https://github.com/apache/cassandra-analytics/blob/cassandra-analytics-$release/CHANGES.txt" >> $mail_file
+echo "[2]: NEWS.txt https://github.com/apache/cassandra-analytics/blob/cassandra-analytics-$release/NEWS.txt" >> $mail_file
+echo "[3]: https://issues.apache.org/jira/browse/CASSANALYTICS" >> $mail_file
+
+
+echo 'Done deploying artifacts. Please make sure to:'
+echo ' 1) "Release" the staging repository from repository.apache.org'
+echo ' 2) wait for the artifacts to sync at https://downloads.apache.org/cassandra/'
+echo ' 3) update the website (TODO provide link)'  # TODO - this is old info and needs updating..
+echo " 4) send announcement email: draft in $mail_file"
+echo ' 5) tweet from @cassandra'
+echo ' 6) release version in JIRA'
+echo ' 7) remove old version (eg: `svn rm https://dist.apache.org/repos/dist/release/cassandra/cassandra-analytics/<previous_version>`)'
+echo ' 8) increment gradle.properties (version) and CHANGES.txt'
+echo ' 9) Add release in https://reporter.apache.org/addrelease.html?cassandra-analytics (same as instructions in email you will receive from the \"Apache Reporter Service\")'
```

Comment:
```suggestion
echo ' 9) Add release in https://reporter.apache.org/addrelease.html?cassandra (same as instructions in email you will receive from the \"Apache Reporter Service\")'
```

top-level project (committee) is still cassandra.

cassandra-analytics will be distinguish when filling out the form, i.e. the `Full version name: ` will be `cassandra-analytics-x.y.z` (instead of just `5.0.4` for C* releases)